docs/source/usage.rst

```diff
@@ -35,12 +35,14 @@ A simple connection with BigchainDB can be established like this.
 
 	const conn = new driver.Connection(API_PATH)
 
-It is also possible to connect to a BigchainDB node of the IPDB test net.
-To do so, you need to pass the **app_id and app_key**.
+Previously you could connect to IPDB testnet, but it has been shutdown.
+Today, you can create an account for the 'BigchainDB Testnet' here_.
+.. _here: https://testnet.bigchaindb.com/?utm_source=3Scale&utm_campaign=21a328d6cd-EMAIL_CAMPAIGN_2018_01_18&utm_medium=email&utm_term=0_b98508b5a9-21a328d6cd-67355241
+To use the tesnet, you need to pass the **app_id and app_key** to get access.
 
 .. code-block:: js
 
-	let bdb = new driver.Connection('https://test.ipdb.io/api/v1/', {
+	let bdb = new driver.Connection('https://test.bigchaindb.com/api/v1/', {
 		app_id: 'dgi829l9',
 		app_key: 'u008ik1bf83b43ce3a95uu0727e66fb9'
 	})
```

Comment:
I guess those credentials were valid for IPDB, not for the BigchainDB Test Network anymore. Maybe they could be avoided with 
```
{
        app_id: "Get one from testnet.bigchaindb.com",
        app_key: "Get one from testnet.bigchaindb.com"
    }
```